### PR TITLE
Feat/80/respond with url

### PR DESCRIPTION
**Pull request by matt-newhall**

### CODE REVIEW

File: src/utils/openai.ts [hunk 1]
```diff
@@ -4,19 +4,31 @@ import fetchContext from '@/utils/fetchContext';
...
   const contextResponse = await fetchContext(embedding);
 
   if (contextResponse.length === 0) {
     console.warn(
       `Warning: Could not find any similar embeddings from the database relating to the question "${userQuestion}".`
     );
-    return 'Unable to find information on this topic.';
+    return {
+      chatMessage: 'Unable to find information on this topic.',
+      associatedUrl: '',
```
Comment (SimonLTheodo) [inline src/utils/openai.ts:24]: What's displayed in this case? An empty link?

File: tailwind.config.cjs [hunk 2]
```diff
@@ -24,6 +24,7 @@ module.exports = {
         x_light_grey: '#dadada',
         grey: '#666666',
         wizz_grey: '#F2F2F2',
+        wizz_pink: '#C6007E',
```
Comment (SimonLTheodo) [inline tailwind.config.cjs:27]: Love it

File: src/utils/openai.ts (hunk 1 above)
Comment (matt-newhall) [reply src/utils/openai.ts:24]: The 'You can find more information at this link' section isn't displayed :)

Comment (SimonLTheodo) [reply src/utils/openai.ts:24]: Perfect

**Review by SimonLTheodo — APPROVED** [review]

**Pull request merged**